core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -87,6 +91,24 @@ angular.module('oppia').component('preferencesPage', {
         _saveDataItem('user_bio', userBio);
       };
 
+      ctrl.checkBioChanged = function() {
+        ctrl.allowreload = false;
+      };
+
+      ctrl.saveBeforeUnload = function() {
+        WindowRef.nativeWindow.addEventListener(
+          'beforeunload', ctrl.confirmBeforeLeaving);
+      };
+
+      ctrl.confirmBeforeLeaving = function(e) {
+        if (!ctrl.allowreload) {
+          // This message is irrelevant, but is needed to trigger the
+          // confirmation before leaving.
+          e.returnValue = 'Sure?';
```

Comment:
Can you please add a ref link in the comment on why this needs to be 'Sure?' and nothing else?

Reply:
If you look at the comment it says:
>           // This message is irrelevant, but is needed to trigger the confirmation before leaving.

So it can be anything not only 'Sure?'

Reply:
Ah, I misunderstood (and assuming can be confusing for other developers maybe?). Can we have an empty string? Also, explain in the code comment that assigning a string to returnValue shows prevent modal in chrome/edge etc.